mlflow/store/file_store.py

```diff
@@ -138,7 +139,18 @@ def list_experiments(self, view_type=ViewType.ACTIVE_ONLY):
             rsl += self._get_active_experiments(full_path=False)
         if view_type == ViewType.DELETED_ONLY or view_type == ViewType.ALL:
             rsl += self._get_deleted_experiments(full_path=False)
-        return [self._get_experiment(exp_id, view_type) for exp_id in rsl]
+        experiments = []
+        for exp_id in rsl:
+            try:
+                experiment = self._get_experiment(exp_id, view_type)
```

Comment:
What is the scenario where this error can happen? When you remove the meta.yaml file from the experiment directory?

Reply:
Yes. Human error. 

`List` and `Search` APIs will log a warning about malformed entity but eat up the exception, if `meta.yaml` is missing (since the intension here to get valid experiments or runs)
`Get`, `Remove`, `Restore` will surface the exception to caller (since the caller intended to do a (c)RUD operation on a entity)

Reply:
Gotcha